client/docs/package-template/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "package-name",
+  "license": "agpl-3.0-or-later",
+  "description": "Short package description.",
+  "version": "1.0.0",
+  "keywords": [
+    "wikijump"
+  ],
+  "scripts": {
+    "clean": "trash dist .tsbuildinfo",
```

Comment:
`trash` is not a universal binary. This should be `rm -rf dist .tsbuildinfo` here.

Reply:
The magic of Lerna, baby! This is actually the `trash-cli` dependency from the root monorepo `package.json`. I made sure specifically _not_ to use native utilities so there wouldn't be Windows <-> Linux issues. Lerna when it bootstraps makes sure to symlink `bin` utilities so that all packages can call them. For example, all packages could call `tsc`.

Reply:
Interesting, I was not aware of this. This seems fine then, since it's designed expressly to be multiplatform.